docs/team/skills.md

```diff
@@ -0,0 +1,103 @@
+to contribute", as well as "ways you can contribute if you have these skills".
+
+### JupyterHub-specific knowledge
+
+- **JupyterHub component architecture**. JupyterHub is the underlying tool of many
+  sub-projects in the JupyterHub/Binder ecosystem, and it's good to undertand its
+  structure. This will make it easier to identify what to work on, what folks are
+  talking about, and what repo might be the place to look at for a given task.
+- **Code structure of the JupyterHub repository**. Familiarizing yourself with the code structure of
+  JupyterHub will help you navigate the repository, and help others to do so as well.
+  There is currently no single place that documents this (if you'd like to create a
+  PR to add it, we'd love to help! [Here's an issue to discuss](https://github.com/jupyterhub/jupyterhub/issues/3080)).
+- **Traitlets configuration**. JupyterHub uses [a tool called traitlets](https://traitlets.readthedocs.io/en/stable/)
+  for configuration. You should familiarize yourself with Traitlets as well as the
+  specific configuration values for JupyterHub. The traitlets documentation could be
+  improved, if you're interested in helping out, [see this issue](https://github.com/jupyterhub/team-compass/issues/300).
+- **YAML Configuration**. JupyterHub uses YAML, a way of structuring text as configuration
+  files, in order to control the behavior of many tools in the JupyterHub ecosystem (as
+  well as to configure CI/CD infrastructure). [Here is a quick primer on YAML](https://github.com/darvid/trine/wiki/YAML-Primer).
+- **Testing infrastructure**. JupyterHub projects generally use [pytest](https://docs.pytest.org/en/stable/)
+  for testing. However, some projects involve more complex tests than you'd normally find
+  in a Python project. For example, mocking HTTP servers, or infrastructure that runs in the cloud.
+
+### Server infrastructure
+
+- **HTTP and REST APIs**. These are the ways in which services, extensions, and some
+  authenticators interact with a JupyterHub.
+- **tornado / asyncio**. JupyterHub uses these tools in order to send messages across
+  JupyterHub infrastructure (e.g., between JupyterHub processes and user servers).
+
+### UI/UX
+
+- **HTML, CSS, and Jinja**. HTML and CSS are used to generate user-facing pages throughout
+  JupyterHub. Jinja is a templating engine that allows you to create HTML pages with
+  variables inserted into them for customization.
+- **Javascript**. Some dynamic content on JupyterHub pages (or via extensions) uses
+  Javascript to control the page's behavior.
+* **UX/UI knowledge**. Aside from specific *implementations* of JupyterHub interfaces
+  (which generally use HTML), it is also helpful to know about the structure and design
+  of user interfaces, and considering the user experience as well.
+
+### Deployment Infrastructure
+
+- **Docker**. JupyterHub uses Docker throughout its projects to provide environments that
```

Comment:
```suggestion
- **Python**. Most JupyterHub projects are distributed as Python packages on [Pypi](https://pypi.org/). Packages are often installed in [virtual Python environments](https://docs.python-guide.org/dev/virtualenvs/) or [Conda environments](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
- **Docker**. JupyterHub uses Docker throughout its projects to provide environments that
```
To me this sounds like you need to know Docker or Kubernetes to deploy JupyterHub, when plain Python environments are equally valid are a lot easier to develop in..

Reply:
PyPI